packages/apps-config/src/extensions/index.ts

```diff
@@ -10,22 +14,66 @@ interface Extension {
   name: string;
 }
 
-// The list of known extensions including the links to tem on the store. This is used when
-// no extensions are actually available, promoting the user to install one or more
+export interface Browsers {
+  chrome: string;
+  firefox: string;
+}
+
+export interface RawExtension {
+  browsers: Browsers;
+  desc: string;
+  name: string;
+}
+
+export interface PreppedExtension extends RawExtension {
+  isInstalled: boolean;
+}
 
-export const availableExtensions: Record<Browser, Extension[]> = [
+export const extensionList: RawExtension[] = [
   {
     browsers: {
       chrome: 'https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd',
       firefox: 'https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/'
     },
     desc: 'Basic account injection and signer',
-    name: 'polkadot-js extension'
+    name: 'polkadot-js'
+  },
+  {
+    browsers: {
+      chrome: 'https://chrome.google.com/webstore/detail/talisman-wallet/fijngjgcjhjmmpcmkeiomlglpeiijkld',
+      firefox: 'https://addons.mozilla.org/en-US/firefox/addon/talisman-wallet-extension/'
+    },
+    desc: 'Talisman is a Polkadot wallet that unlocks a new world of multichain web3 applications in the Paraverse',
+    name: 'talisman'
   }
-].reduce((available: Record<Browser, Extension[]>, { browsers, desc, name }): Record<Browser, Extension[]> => {
+];
+
+// The list of known extensions including the links to them on the store. This is used when
+// no extensions are actually available, promoting the user to install one or more
+
+export const availableExtensions: Record<Browser, Extension[]> = extensionList.reduce((available: Record<Browser, Extension[]>, { browsers, desc, name }): Record<Browser, Extension[]> => {
   Object.entries(browsers).forEach(([browser, link]): void => {
     available[browser as 'chrome'].push({ desc, link, name });
   });
 
   return available;
 }, { chrome: [], firefox: [] });
+
+export const supportedExtensionsNames = () => extensionList.map(({ name }) => name);
+
+export const getUserInstalledExtensions = () => {
```

Comment:
```suggestion
export function supportedExtensionsNames (): string[] {
  return extensionList.map(({ name }) => name);
}

export function getUserInstalledExtensions (): <....type-to-be-added...> {
```

Reply:
Prefer `function` as opposed to shortcuts.

Also ensure that each function has an explicit return, there are some edge-cases that popup every now and again where lack of return types end up with some "small" head-scratchers.

Reply:
> Prefer `function` as opposed to shortcuts.

May I ask you, what's the reason behind this?

Reply:
This is for the sake of consistency - there is one way to write code so there is no overhead in thinking when maintaining it. So doesn't matter which file you touch, it looks and feels the same.